Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptWrapperAddController.java

```diff
@@ -83,9 +84,13 @@ private void initBinder(WebDataBinder binder) {
      */
     @RequestMapping(value = "auth/conceptlist/addconceptwrapper")
     public String prepareConceptWrapperAdd(@ModelAttribute ConceptWrapperAddBean conceptWrapperAddBean,
-            HttpServletRequest req, ModelMap model) {
+            HttpServletRequest req, ModelMap model) throws LuceneException, IllegalAccessException, IndexerRunningException{
         model.addAttribute("types", conceptWrapperService.fetchAllConceptTypes());
         model.addAttribute("lists", conceptWrapperService.fetchAllConceptLists());
+        String wrapperId = conceptWrapperAddBean.getWrapperids();
+        ConceptEntry entry = conceptManager.getWordnetConceptEntry(wrapperId);
+        conceptWrapperAddBean.setDescription(entry.getDescription());
+        conceptWrapperAddBean.setWord(entry.getWord());
```

Comment:
what about concept list and pos?

Reply:
Have set the concept list and pos as well